### PR TITLE
darktile: 0.0.10 -> 0.0.11

### PR DESCRIPTION
**Pull request by flexagoon**
###### Description of changes

Updates darktile to 0.0.11 (#220883)

Makes @sogreatandpowerfull the maintainer (https://github.com/NixOS/nixpkgs/issues/220883#issuecomment-1465710233)

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sogreatandpowerfull** [comment]
It works fine!

File: pkgs/applications/terminal-emulators/darktile/default.nix [hunk 1]
```diff
@@ -60,6 +60,6 @@ stdenv.mkDerivation rec {
     downloadPage = "https://github.com/liamg/darktile/releases";
     changelog = "https://github.com/liamg/darktile/releases/tag/v${version}";
     license = licenses.mit;
-    maintainers = with maintainers; [ flexagoon ];
+    maintainers = with maintainers; [ sogreatandpowerfull ];
```
Comment (sogreatandpowerfull) [inline pkgs/applications/terminal-emulators/darktile/default.nix:63]: @flexagoon can you change it to `sogreatandpowerful`

Comment (flexagoon) [reply pkgs/applications/terminal-emulators/darktile/default.nix:63]: Sure, I'll change it

**Comment by sogreatandpowerfull** [comment]
Thank you!

File: pkgs/applications/terminal-emulators/darktile/default.nix (hunk 1 above)
Comment (sogreatandpowerfull) [reply pkgs/applications/terminal-emulators/darktile/default.nix]: Thank you!

**Review by ghost — APPROVED** [review]
LGTM!

**Comment by NickCao** [comment]
@ofborg build darktile

**Review by Janik-Haag — COMMENTED** [review]
When trying to build this locally it fails with this error:
```
unpacking source archive /build/v0.0.11.tar.gz
building '/nix/store/hblg3h8af892lhfajznx0wb1h1py5rzb-darktile-0.0.11.drv'...
unpacking sources
unpacking source archive /nix/store/g86grmn1dlm8chpc97szc85vf4gq5lqx-source
source root is source
patching sources
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/5ynbf6wszmggr0abwifdagrixgnya5vy-bash-5.2-p15/bin/bash HOME=\$TMP
./scripts/build.sh
/nix/store/5ynbf6wszmggr0abwifdagrixgnya5vy-bash-5.2-p15/bin/bash: line 1: ./scripts/build.sh: cannot execute: required file not found
make: *** [Makefile:6: build] Error 127
```
:confused:

**Comment by Janik-Haag** [comment]
@flexagoon are you still interested?

**Comment by Janik-Haag** [comment]
I'm closing this for now, feel free to reopen once you have time to work on it again.

**Pull request closed without merging**